wg-apply/README.md

```diff
@@ -8,6 +8,8 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 --->
 # Apply Working Group
 
+Superseded by [wg-api-expression](/wg-api-expression/README.md).
```

Comment:
Would it make sense for WG Apply to move to https://github.com/kubernetes/community/tree/master/archive? The content would still be preserved that way and we won't have inactive WGs in the root of the repo.

Reply:
That's exactly what we were looking for I think, thanks!